usecases/config/config_handler.go

```diff
@@ -356,6 +398,22 @@ func (f *WeaviateConfig) parseConfigFile(file []byte, name string) (Config, erro
 	return config, nil
 }
 
+// fromFlags parses values from flags given as parameter and overrides values in the config
+func (f *WeaviateConfig) fromFlags(flags *Flags) {
+	if flags.RaftPort > 0 {
+		f.Config.Raft.Port = flags.RaftPort
+	}
+	if flags.InternalRPCPort > 0 {
```

Comment:
in order to be consistent with the naming I would also rename the flags to have the `Raft` prefix: `flags.RaftInternalRPCPort`